### PR TITLE
README and leaderboard changes

### PR DESCRIPTION
**Pull request by CompuGenius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 -   `BOT_TOKEN` - your Discord bot token
 -   `CLIENT_ID` - your Discord bot application ID (used for the invite link) (optional)
+-   `BOT_PREFIX` - your preferred bot prefix if not using slash commands (default is `2!`)
```
Comment (vrumger) [inline README.md:7]: Please sort this alphabetically.

File: commands/leaderboard.js [hunk 2]
```diff
@@ -18,23 +18,49 @@ module.exports = {
                 guildName: channel.guildName,
```
Comment (vrumger) [inline commands/leaderboard.js:18]:
```suggestion
                guildId: channel.guildId,
                guildName: channel.guildName,
```

File: commands/leaderboard.js [hunk 3]
```diff
@@ -18,23 +18,49 @@ module.exports = {
...
+            let score = 0;
+
+            for (let i = 0; i < guilds.length; i++) {
+                let guild = guilds[i];
+                if (guild.guildName === interaction.guild.name) {
+                    score = guild.score;
+                    index = i + 1;
+                    break;
+                }
+            }
```
Comment (vrumger) [inline commands/leaderboard.js:52]:
```suggestion
            const index = guilds.findIndex(guild => guild.guildId === interaction.guildId);
            const { score } = guilds[index];
```

File: commands/leaderboard.js [hunk 4]
```diff
@@ -18,23 +18,49 @@ module.exports = {
...
+        const topGuilds = guilds.slice(0, 10);
 
         const embed = new Discord.MessageEmbed();
 
+        let description = topGuilds
+            .map((guild, index) =>
+                guild.guildName === interaction.guild.name
+                    ? `**${index + 1}. ${
+                          guild.guildName
+                      } - ${new Intl.NumberFormat().format(guild.score)}**`
```
Comment (vrumger) [inline commands/leaderboard.js:31]: This can probably be a function.

**Review by vrumger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CompuGenius) [reply README.md:7]: I put them in order logically. Up to you of you want to sort.

File: commands/leaderboard.js (hunk 4 above)
Comment (CompuGenius) [reply commands/leaderboard.js:31]: Feel free to do so.

**Review by vrumger — APPROVED** [review]

**Pull request merged**